etc/kayobe/users.yml

```diff
@@ -20,15 +21,8 @@ users_default:
   ssh_key:
   # Authorise all users' SSH keys for the kolla user. This allows them to
   # operate kolla.
-  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_bharat.pub') }}"
-  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_isaac.pub') }}"
-
-- username: isaac
```

Comment:
Did you mean to delete @Wasaac's account? It is probably better practice not to share accounts.

Reply:
I thought it might have been added here by accident. Does it really make sense to have an account per user in all the overcloud and seed machines?

Reply:
Anyone who wants to run kayobe needs to have their key authorized for the stack and kolla users.

Reply:
The keys are intact for stack and kolla users